Content.Shared/Borgs/LawsComponent.cs

```diff
@@ -0,0 +1,34 @@
+using Robust.Shared.GameStates;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.Borgs
+{
+    [RegisterComponent, NetworkedComponent]
+    public sealed class LawsComponent : Component
+    {
+        [DataField("laws")]
+        public List<string> Laws = new List<string>();
+
+        [DataField("canState")]
+        public bool CanState = true;
+
+        /// <summary>
+        ///     Antispam.
+        /// </summary>
+        public TimeSpan? StateTime = null;
+
+        [DataField("stateCD")]
+        public TimeSpan StateCD = TimeSpan.FromSeconds(30);
```

Comment:
Stating laws should not be in here. Make a CurrentlyStatingLawsComponent or similar that is removed on finish.

Reply:
isn't that adding an entityquery for something that's just a cooldown

Reply:
Does it matter?